daily_milestone.go

```diff
@@ -83,22 +85,37 @@ func getProjectID(baseURL string, Token string, Projectname string, Namespace st
 	return strconv.Itoa(project.ID)
 }
 
-// CreateMilestoneData is the Function similar to Create_Milestone_Data in the Python implementation
-func CreateMilestoneData(Advance int) []string {
+// CreateMilestoneData is used to check the due date using the time package of python
+func createMilestoneData(advance int) []string {
 	today := time.Now().Local()
-	i := 0
 	list := []string{}
-	for i < Advance {
+	for i := 0; i < advance; i++ {
 		date := today.AddDate(0, 0, i)
 		dateiso := date.Format("2009-01-02")                            // To Format to ISOFormat and it converts to string so can be used in list directly
 		datelist := []string{"Title: ", "dateiso", "due_date", dateiso} // Was unable to get a map in a list so made this
 		y := strings.Join(datelist, ",")
 		list = append(list, y)
-		i++
+
 	}
 	return list
 }
 
+func createMilestones(baseURL string, token string, projectID string, milestones []string) string {
+	strurl := []string{"https://", baseURL, "/projects", projectID, "/milestones"}
+	url := strings.Join(strurl, ",")
+
+	for _, m := range milestones {
+		req, err := http.NewRequest("POST", url, nil)
+		if err != nil {
+			logger.Println(err)
+			break
+		}
+		req.Header.Add("", m)
+		req.Header.Add("Private-Token", token)
+
```

Comment:
Still missing the request sending.

Reply:
Hey can you please tell me what is request sending?

Reply:
You do prepare the request, but you never send it.
Something like this is still missing: ```resp, err := client.Do(req)```

Reply:
Thanks a lot.